json-endpoints/jetpack/class.jetpack-json-api-sync-endpoint.php

```diff
@@ -40,6 +40,8 @@ protected function result() {
 		if ( empty( $modules ) ) {
 			$modules = null;
 		}
+
```

Comment:
Instead of doing this. Should we have a remote (more universal way) to unhook the sender? 
Maybe by passing a query parameter to the url. 

This way if the parameter is present we could be more intentional about any request and not even hook it if the parameter is present in the request.

Reply:
I like the sound of it.